src/main/java/com/Saurabh/GitHubAiAPI/StringManipulation.java

```diff
@@ -45,4 +49,12 @@ public static String findVowels(String input) {
         }
         return vowels.toString();
     }
```

Comment:
The file should end with a newline to comply with the POSIX standard and to prevent any unexpected behavior in certain command line tools.